tests/conftest.py

```diff
@@ -17,24 +18,45 @@ def is_responsive(url):
 
 
 @pytest.fixture(scope="session")
-def notebook_service(docker_ip, docker_services):
-    """Ensure that HTTP service is up and responsive."""
+def docker_compose(docker_services):
+    return docker_services._docker_compose
 
-    docker_compose = docker_services._docker_compose
 
-    # assurance for host user UID other that 1000
-    chown_command = "exec -T -u root aiidalab bash -c 'chown -R jovyan:users /home/jovyan/apps/aiidalab-qe'"
-    docker_compose.execute(chown_command)
+@pytest.fixture(scope="session")
+def aiidalab_exec(docker_compose):
+    def execute(command, user=None, workdir=None, **kwargs):
+        opts = "-T"
+        if user:
+            opts = f"{opts} --user={user}"
```

Comment:
```suggestion
            opts = f"-T --user={user}"
```
I mean this, I think it is more clear, no?

Reply:
But I need to support the case when both `workdir` and `user` are None.

Reply:
I see, I'd prefer to create a list and construct the options using `' '.join(options_list)`. But your approach is also good.